krystal-visualization/src/main/java/com/flipkart/krystal/visualization/models/AnnotationInfo.java

```diff
@@ -0,0 +1,12 @@
+package com.flipkart.krystal.visualization.models;
+
+import java.util.Map;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class AnnotationInfo {
```

Comment:
As discussed use `toString` instead of reflection and this class. Then this class can be removed